bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -124,7 +124,7 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
 
         final SPInstObsComp inst = getContextInstrumentDataObject();
         _w.newMenu.setEnabled(enabled && inst != null);
-        _w.tag.setEnabled(enabled && !selectionIsBasePosition());
+        _w.tag.setEnabled(enabled && selectedGroup().isEmpty() && !selectionIsAutoTarget() && !selectionIsBasePosition());
```

Comment:
That's right.
I had a merge conflict and resolved it, but now am having an issue when I try to commit where I get Untracked files: BagsManager.CacheSize, and a bunch of other things. (This has happened before after rebasing / resolving merge conflicts.)

As this is just one file being changed, I'm going to copy it, open a new branch, and a new PR to work around this so we can get the build going. If you know why this happens, we can discuss it later.